### PR TITLE
Chartheight

### PR DESCRIPTION
**Pull request by cindylovescoding**
Fix tooltip overflow

### CODE REVIEW

**Comment by cindylovescoding** [comment]
Previously the tooltip gets cut because of the default highchart-container attribute:
image

Fix that in this PR:


image

File: AngularApp/projects/diagnostic-data/src/lib/components/highcharts-graph/highcharts-graph.component.ts [hunk 1]
```diff
@@ -200,7 +200,8 @@ export class HighchartsGraphComponent implements OnInit {
             tooltip: {
                 shared: true,
                 enabled: true,
-                valueDecimals: 2
+                valueDecimals: 2,
+                useHTML: true,
```
Comment (SharmaAjay19) [inline AngularApp/projects/diagnostic-data/src/lib/components/highcharts-graph/highcharts-graph.component.ts]: Can we add an opacity setting here to resolve this 
image

Comment (puneetg1983) [reply AngularApp/projects/diagnostic-data/src/lib/components/highcharts-graph/highcharts-graph.component.ts]: +1 on this

Comment (cindylovescoding) [reply AngularApp/projects/diagnostic-data/src/lib/components/highcharts-graph/highcharts-graph.component.ts]: the background above the axis is not coming through. Changing opacity won't work

File: AngularApp/projects/applens/src/styles.scss [hunk 2]
```diff
@@ -157,3 +157,16 @@ body {
...
   min-height: 70vh !important;
 }
+
+/* highchart container configuration */
+.highcharts-container {
+    overflow: visible !important;
+  }
+
+.highcharts-container svg {
+    overflow: visible !important;
```
Comment (puneetg1983) [inline AngularApp/projects/applens/src/styles.scss]: Just wanted to confirm - will we get a vertical scrollbar if there are double the number of legends as shown in the screenshot or will the height of the legend tooltip keep increasing

**Comment by ShekharGupta1988** [comment]
> Previously the tooltip gets cut because of the default highchart-container attribute:
> image
> 
> Fix that in this PR:
> 
> image

Very nice :)

File: AngularApp/projects/applens/src/styles.scss (hunk 2 above)
Comment (cindylovescoding) [reply AngularApp/projects/applens/src/styles.scss]: I think it will keep increasing just like what we had before in nvd3.

**Pull request merged**